src/main/java/frc/team2412/robot/commands/diagnostic/LauncherDiagnosticCommand.java

```diff
@@ -0,0 +1,22 @@
+package frc.team2412.robot.commands.diagnostic;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.team2412.robot.commands.launcher.SetAngleCommand;
+import frc.team2412.robot.commands.launcher.SetLaunchSpeedCommand;
+import frc.team2412.robot.commands.launcher.StopLauncherCommand;
+import frc.team2412.robot.subsystems.LauncherSubsystem;
+
+public class LauncherDiagnosticCommand extends SequentialCommandGroup {
+	private final LauncherSubsystem launcherSubsystem;
+	private final double Angle;
+
+	public LauncherDiagnosticCommand(LauncherSubsystem launcherSubsystem) {
+		this.launcherSubsystem = launcherSubsystem;
+		this.Angle = launcherSubsystem.getAngle();
+		addCommands(
+				new SetAngleCommand(launcherSubsystem, 45),
+				new SetAngleCommand(launcherSubsystem, Angle),
```

Comment:
If the launcher starts at 45 degress, these two end up doing  nothing. I'd recommend adding a line to verify the launcher can reach the amp position (can use 90 degrees for that), in addition to 45 degrees and then resetting to where it started. Even though `SetAngleCommand` won't finish until the target angle is reached, it'd be best to add a short pause between these (2 or 3 seconds) so it's easy to see if the launcher reaches the expected angle each time.

Reply:
Done, ty!